src/data/services/ZendeskApiService.js

```diff
@@ -0,0 +1,25 @@
+import apiClient from '../apiClient';
+import { configuration } from '../../config';
+
+class ZendeskApiService {
+    static baseUrl = configuration.LMS_BASE_URL;
+    static createZendeskTicket(options) {
+      const url = `${ZendeskApiService.baseUrl}/zendesk_proxy/v1`;
```

Comment:
I would just use configuration.LMS_BASE_URL instead of re-declaring it in the class